lib/actions/ssh.coffee

```diff
@@ -92,7 +92,7 @@ module.exports =
 			Promise.props
 				username: resin.auth.whoami()
 				uuid: device.uuid		# get full uuid
-				containerId: resin.models.device.getApplicationInfo(device.uuid).get('containerId')
+				containerId: resin.models.device.getApplicationInfo(device.uuid).then(JSON.parse).get('containerId')
```

Comment:
@lekkas Very interesting indeed. It's not that resin-request doesn't parse JSON any more: hitting http://echo.jsontest.com/key/value with resin-request gives me a JS object back, but hitting `https://api.resin.io/supervisor/v1/apps/<my-app-id>` gives me a string.

Doing some more testing, the problem seems to be the content-type headers. If I hit `https://api.resin.io/supervisor/v1/apps/<my-app-id>`, it comes back with a json body, but `Content-Type: text/html`. This isn't the case for other endpoints: `https://api.resin.io/v1/application` comes back with `application/json` correctly for example, and resin-request returns a parsed object body from that.

This definitely seems broken, and we should fix in the API. The previous version of the SDK does still work, I'd guess because the previous version of resin-request ignored the content type header, and maybe it just parsed every response? We could do something like that ourselves too as a quick fix, but I'd much rather fix this in the API properly if that's easy.